MoltenVK/MoltenVK/GPUObjects/MVKDeviceFeatureStructs.def

```diff
@@ -64,7 +64,9 @@ MVK_DEVICE_FEATURE(VariablePointer,                   VARIABLE_POINTER,
 MVK_DEVICE_FEATURE(VulkanMemoryModel,                 VULKAN_MEMORY_MODEL,                    3)
 MVK_DEVICE_FEATURE_EXTN(FragmentShaderBarycentric,    FRAGMENT_SHADER_BARYCENTRIC,     KHR,   1)
 MVK_DEVICE_FEATURE_EXTN(PortabilitySubset,            PORTABILITY_SUBSET,              KHR,  15)
+MVK_DEVICE_FEATURE_EXTN(ShaderTerminateInvocation,    SHADER_TERMINATE_INVOCATION,     KHR,   1)
 MVK_DEVICE_FEATURE_EXTN(4444Formats,                  4444_FORMATS,                    EXT,   2)
+MVK_DEVICE_FEATURE_EXTN(DepthClipControl,             DEPTH_CLIP_CONTROL,              EXT,   1)
```

Comment:
Bah. Maybe I shouldn't have the right to merge changes...

Reply:
It would be nice to figure out a compile-time assertion that could compare that case statements in `getFeatures()` to `MVKDeviceFeatureStructs.def`, but it would be a challenge at best, and maybe not even possible.